### PR TITLE
handle accpet bid error - https://particl.atlassian.net/browse/PD-151

### PR DESCRIPTION
**Pull request by vikas-cis**
#1036

### CODE REVIEW

**Comment by HenkSwardt** [comment]
https://particl.atlassian.net/browse/PD-151

File: src/app/core/market/api/bid/bid.service.ts [hunk 1]
```diff
@@ -55,7 +50,14 @@ export class BidService {
 
   acceptBidCommand(hash: string, id: number): Observable<any> {
     const params = ['accept', hash, id];
-    return this.market.call('bid', params);
+    return new Observable((observer) => {
```
Comment (kewde) [inline src/app/core/market/api/bid/bid.service.ts]: Why a new observable? It's already an observable..

File: src/app/core/market/api/bid/bid.service.ts [hunk 2]
```diff
@@ -55,7 +50,12 @@ export class BidService {
 
   acceptBidCommand(hash: string, id: number): Observable<any> {
     const params = ['accept', hash, id];
-    return this.market.call('bid', params);
+    return this.market.call('bid', params).do(res => console.log(res), (error) => {
```
Comment (kewde) [inline src/app/core/market/api/bid/bid.service.ts]: .catch()?

Comment (kewde) [reply src/app/core/market/api/bid/bid.service.ts]: Check the Rxjs docs.

**Pull request merged**